Riot/Modules/Authentication/SSO/SSOURLConstants.swift

```diff
@@ -25,6 +25,5 @@ enum SSOURLConstants {
     
     enum Paths {
         static let redirect = "/_matrix/client/r0/login/sso/redirect"
```

Comment:
We are missing a slash at the end:
```suggestion
        static let redirect = "/_matrix/client/r0/login/sso/redirect/"
```